join.spec.md

```diff
@@ -187,9 +181,9 @@ Join a field with a particular subgraph.
 
 ```graphql definition
 directive @join__field(
```

Comment:
This section doesn't document what requires and provides are at all, nor do they show up in examples.

Reply:
Fixed (no examples though)